geomesa-hbase/geomesa-hbase-gs-plugin/pom.xml

```diff
@@ -57,6 +57,18 @@
             <version>${hbase.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hbase</groupId>
+            <artifactId>hbase-hadoop-compat</artifactId>
+            <version>${hbase.version}</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hbase</groupId>
+            <artifactId>hbase-hadoop2-compat</artifactId>
+            <version>${hbase.version}</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
these are actually only needed for hbase 1.4 right? i think we can remove them

Reply:
Removed